stock_by_warehouse_sale/models/sale_order.py

```diff
@@ -5,11 +5,8 @@ class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
     warehouses_stock = fields.Text(store=False, readonly=True)
-    warehouse_id = fields.Many2one(
```

Comment:
remove it because 
https://github.com/odoo/odoo/blob/14.0/addons/sale_stock/models/sale_order.py#L275